library/Rules/AlwaysInvalid.php

```diff
@@ -13,8 +13,16 @@
 
 namespace Respect\Validation\Rules;
 
-class AlwaysInvalid extends AbstractRule
+/**
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
```

Comment:
Can you add an line before telling what this rule does? You will find that in [AlwaysInvalid.md](https://github.com/Respect/Validation/blob/master/docs/AlwaysInvalid.md)